### PR TITLE
adapting JS to the new API

### PR DESCRIPTION
**Pull request by vinyll**

### CODE REVIEW

File: api/views.py [hunk 1]
```diff
@@ -1,26 +1,39 @@
...
+from user.models import User
 from core import app
 from around.models import Spot
 
 
-@app.route('/api/spots')
-def spots():
-    spots = Spot.objects.all().to_json()
-    return Response(spots, mimetype='application/json')
+@app.route('/api/spots/')
```
Comment (fuentesloic) [inline api/views.py]: why you add an `/` at the end of the path ?

File: static/js/around-map.js [hunk 2]
```diff
@@ -21,39 +20,42 @@ map.addControl(new GeolocateControlWrapper())
...
+
+  const hash = getHash()
+  if(hash && hash === el.id) {
+    superCard(hash)
+    firstCard(hash)
+    moveTo(poi.location, 11)
+  }
+}
+
+
```
Comment (fuentesloic) [inline static/js/around-map.js]: useless line break

File: static/js/around-map.js [hunk 3]
```diff
@@ -188,5 +190,15 @@ document.querySelector('#poi-form form').addEventListener('submit', event => {
...
+  .then(r => r.json())
+  .then(response => {
+    if (response.id) {
+      alert('success', 'You spot was saved succefully')
+    }
+    else {
+      alert('error', 'Something went wrong. Please try again later.')
+    }
+    document.querySelector('#poi-form').classList.remove('visible')
+
```
Comment (fuentesloic) [inline static/js/around-map.js]: empty line break, call the `addPOI` function

File: api/views.py (hunk 1 above)
Comment (vinyll) [reply api/views.py]: @fuentesloic: https://github.com/neomad-team/neomad.org/pull/151#issue-249887808

**Comment by vinyll** [comment]
@fuentesloic: comments are fixed and branch is rebased

**Pull request merged**